checker/tests/nullness/ConcurrentHashMapTest.java

```diff
@@ -0,0 +1,16 @@
+
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+class Issue1142 {
```

Comment:
In Java, the class name and the file name should match.  Please change one of them.